packages/react-native-sdk/docusaurus/docs/reactnative/05-advanced/02-push-notifications.mdx

```diff
@@ -413,123 +281,88 @@ import { navigationRef } from './src/utils/staticNavigationUtils';
+import { STREAM_API_KEY } from '../../constants';
+import { staticNavigate } from './staticNavigationUtils';
+
+export function setPushConfig() {
+  StreamVideoRN.setPushConfig({
+    ios: {
+      // add your push_provider_name for iOS that you have setup in Stream dashboard
+      pushProviderName: __DEV__ ? 'apn-video-staging' : 'apn-video-production',
+    },
+    android: {
+      // add your push_provider_name for Android that you have setup in Stream dashboard
+      pushProviderName: __DEV__ ? 'firebase-video-staging' : 'firebase-video-production',
+      // configure the notification channel to be used for incoming calls for Android.
+      incomingCallChannel: {
+        id: 'stream_incoming_call',
+        name: 'Incoming call notifications',
+        // This is the advised importance of receiving incoming call notifications.
+        // This will ensure that the notification will appear on-top-of applications.
+        importance: AndroidImportance.HIGH,
+      },
+      // configure the functions to create the texts shown in the notification
+      // for incoming calls in Android.
+      incomingCallNotificationTextGetters: {
+        getTitle: (createdUserName: string) =>
+          `Incoming call from ${createdUserName}`,
+        getBody: (_createdUserName: string) => 'Tap to answer the call',
+      },
+    },
+    // add the callback to be executed a call is accepted, used for navigation
+    navigateAcceptCall: () => {
+      staticNavigate({ name: 'ActiveCallScreen', params: undefined });
+    },
+    // add the callback to be executed when a notification is tapped,
+    // but the user did not press accept or decline, used for navigation
+    navigateToIncomingCall: () => {
+      staticNavigate({ name: 'IncomingCallScreen', params: undefined });
+    },
+    // add the async callback to create a video client 
+    // for incoming calls in the background on a push notification
+    createStreamVideoClient: async () => {
+      // note that since the method is async, 
+      // you can call your server to get the user data or token or retrieve from offline storage.
+      const userId = await AsyncStorage.getItem('@userId');
+      const userName = await AsyncStorage.getItem('@userName');
+      const token = await AsyncStorage.getItem('@userToken');
+      if (!username || !userToken) return undefined;
+      const user = { id: userId, name: userName, token };
+      return new StreamVideoClient({
+        apiKey: STREAM_API_KEY, // pass your stream api key
+        user,
+        token,
+      })
+    },
+  });
 };
 ```
 
-### Listen to callkeep events
+Now call the method outside of your application cycle. That is, alongside your `AppRegistry.registerComponent()` method call at the entry point of your application code. This is because the app can be opened from a dead state through a push notification and in that case, we need to use the config as soon as the JS bridge is initialized. Following is an example,
```

Comment:
```suggestion
Now, call the method outside of your application cycle. That is, alongside your `AppRegistry.registerComponent()` method call at the entry point of your application code. This is because the app can be opened from a dead state through a push notification and in that case, we need to use the config as soon as the JS bridge is initialized. Following is an example,
```